JunitHomework/src/main/java/string/utility/NumbersModifierNotOk.java

```diff
@@ -0,0 +1,34 @@
+package string.utility;
+
+public class NumbersModifierNotOk {
+    public String before;
+
+    public NumbersModifierNotOk() {
+
+    }
+
+    public String getBefore() {
+        return before;
+    }
+
+    public void setBefore(String before) {
+        this.before = before;
+    }
+
+    public String modifyNumbers() {
+        String after;
```

Comment:
again - this naming is not meaningful. We create classes to used them in multiple place.

Also I think here would be enough to just have a return.
Please remove String after.

And for the method naming I think it would be better to replace it with public String replaceTextWithNumber(String inputText)